pkg/apis/troubleshoot/v1beta2/collector_shared.go

```diff
@@ -33,12 +33,13 @@ type Secret struct {
 }
 
 type ConfigMap struct {
-	CollectorMeta `json:",inline" yaml:",inline"`
-	Name          string   `json:"name,omitempty" yaml:"name,omitempty"`
-	Selector      []string `json:"selector,omitempty" yaml:"selector,omitempty"`
-	Namespace     string   `json:"namespace,omitempty" yaml:"namespace,omitempty"`
-	Key           string   `json:"key,omitempty" yaml:"key,omitempty"`
-	IncludeValue  bool     `json:"includeValue,omitempty" yaml:"includeValue,omitempty"`
+	CollectorMeta  `json:",inline" yaml:",inline"`
+	Name           string   `json:"name,omitempty" yaml:"name,omitempty"`
+	Selector       []string `json:"selector,omitempty" yaml:"selector,omitempty"`
+	Namespace      string   `json:"namespace,omitempty" yaml:"namespace,omitempty"`
+	Key            string   `json:"key,omitempty" yaml:"key,omitempty"`
+	IncludeValue   bool     `json:"includeValue,omitempty" yaml:"includeValue,omitempty"`
+	CollectAllData bool     `json:"collectAllData,omitempty" yaml:"collectAllData,omitempty"`
```

Comment:
naming question. whats you thinking behind `collectAllData `? since we already have `includeValue` should we call this `includeAllData`?

Reply:
yeah I think includeAllData is more inline with our convention!